cmd/snap-repair/cmd_run.go

```diff
@@ -85,19 +92,22 @@ func (c *cmdRun) Execute(args []string) error {
 		return err
 	}
 
-	for {
-		repair, err := run.Next("canonical")
-		if err == ErrRepairNotFound {
-			// no more repairs
-			break
-		}
-		if err != nil {
-			return err
-		}
+	for _, rootRepairBrandID := range rootBrandIDs {
+		for {
+			repair, err := run.Next(rootRepairBrandID)
+			if err == ErrRepairNotFound {
+				// no more repairs
+				break
```

Comment:
Shouldn't this continue with the outer loop over rootBrandIDs?

Reply:
Hmm, I'm not sure I understand the question, if we encounter that there are no more repairs for the given brand ID, we should exit the inner loop to try the next brand ID until that brand ID returns no more repairs etc.